package.json

```diff
@@ -11,9 +11,9 @@
 		"email": "admin@tri.be"
 	},
 	"browserslist": [
-		"last 2 versions",
-		"not op_mini all",
-		"not dead"
+		"defaults",
+		"not ios < 17.5",
```

Comment:
Just curious why you decided to include this? It looks like without it we are included some 16.x versions of Safari? I just wonder if it is ok to keep those so we don't have to remember to keep updating this line as things move forward.

Reply:
Good question. Correct. The "defaults" key still supports 16.6, 16.7, and 15.6-8. We don't want or need to support those really old versions of Safari as they often lack features we want to use.

The usage on those versions is minimal; and, since those versions are [no longer supported by Apple](https://en.wikipedia.org/wiki/Safari_(web_browser)#Version_compatibility), I don't see a reason to keep them.